### PR TITLE
Add 2K Games

### PR DESCRIPTION
**Pull request by soodaayush**

### CODE REVIEW

File: img/2/2k.com.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...op-color="#ee2722"/><stop offset="1" stop-color="#f46a4f"/></linearGradient></defs><path fill="#ee2722" d="M0 0c-.644-1.801-2.958-3.265-5.172-3.265h-99.83c-2.213 0-3.485 1.464-2.853 3.265l30.707 81.395c.648 1.801 2.963 3.266 5.176 3.266h95.884c2.209 0 3.493-1.465 2.858-3.266L0 0" clip-path="url(#c)" transform="matrix(3.70509 0 0 -3.70509 400.213 313.677)"/><path fill="url(#d)" d="M531.79 431.046c5.635 0 3.712-5.504 3.712-5.504l-11.588-33.038s-14.63-20.847-52.414-20.847c-37.798 0-49.159 18.507-49.159 18.507s9.922 27.131 12.287 33.561c2.381 6.437 10.792 6.64 10.792 6.64l86.37.681" clip-path="url(#e)" transform="matrix(3.70509 0 0 -3.70509 -1496.8 1602.24)"/><path fill="#fff" d="M423.581 147.078h-82.438l-75.55 55.562 17.869-55.562H151.557c5.754-5.924 14.168-9.537 28.574-12.034l60.44-10.693c24.088-4.272 58.808-16.34 68.393-46.273 14.065-44.053-31.6-53.16-83.372-53.16-40.189 0-94.535 7.265-116.644 61.548h68.13c8.384-20.455 26.347-22.041 43.586-22.041 14.78 0 24.743 3.65 21.827 12.69-2.104 6.61-7.733 10.693-16.165 12.282l-49.749 10.008c-40.315 8.173-84.587 26.54-105.106 87.18h137.096l-37.71 117.926h62.186l13.516-42.308 26.284-18.115 23.68 60.423h82.66l-45.399-96.57 89.797-60.863"/></svg>
```
Comment (Carlgo11) [inline img/2/2k.com.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" style="background:#ea2228;fill:#fff" viewBox="0 0 1000 1000"><path d="M837 465H679L534 570l34-105H316c11-11 27-18 54-22l116-21c46-8 113-31 131-87 27-84-60-101-160-101-77 0-181 14-223 116h130c16-38 51-41 84-41 28 0 47 7 42 24-4 12-15 20-31 23l-96 19c-77 15-162 50-201 165h263l-73 223h120l26-80 50-34 45 114h159l-87-183Z"/></svg>
```

File: entries/2/2k.com.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "2K Games": {
+    "domain": "2k.com",
+    "contact": {
+      "form": "https://support.2k.com/",
```
Comment (Carlgo11) [inline entries/2/2k.com.json:5]:
```suggestion
      "form": "https://support.2k.com/",
      "facebook": "2k",
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Comment by soodaayush** [comment]
Changes made.

**Review by Carlgo11 — APPROVED** [review]
Thank you! 🙂
Your contribution will be visible in around 10 minutes. Sometimes it takes longer due to server and browser cache.

**Pull request merged**